plugins/com.google.cloud.tools.eclipse.sdk/.options

```diff
@@ -0,0 +1,4 @@
+# Uncomment for development. To be loaded in a hosted debug Eclipse instance,
+# open the "gcloud-eclipse-tools" run configuration, go to the "Tracing" tab,
+" and check "Enable tracing" and "com.google.cloud.tools.eclipse.sdk".
+#com.google.cloud.tools.eclipse.sdk/managed.cloud.sdk=true
```

Comment:
If you don't have this uncommented then it won't show up in the tracing tab.  It should be set to `false` to reflect the default.